Add sample encrypted container image instructions

stevenhorsman opened the pull request:
Add instructions for how to set-up, create and validate creating a workload from the sample encrypted container image

Fixes: #confidential-containers/operator#77
Signed-off-by: stevenhorsman <steven@uk.ibm.com>

stevenhorsman committed:
Add sample encrypted container image instructions

Add instructions for how to set-up, create and validate creating a
workload from the sample encrypted container image

Fixes: #confidential-containers/operator#77
Signed-off-by: stevenhorsman <steven@uk.ibm.com>

bpradipt commented on quickstart.md:
```diff
@@ -174,13 +174,85 @@ root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/contain
 
-Change the yaml to point to the sample container image
-The keys for this image should be in the rootfs
+We have prepared a sample CoCo operator custom resource that is based on the standard `ccruntime.yamlccruntime.yaml`, but in addition has the the decryption keys and configuration required to decrypt this sample container image. To test out creating a workload from the sample encrypted container image, we can take the following steps:
```
nit: `ccruntime.yamlccruntime.yaml` => `ccruntime.yaml`

bpradipt commented on quickstart.md:
```diff
@@ -174,13 +174,85 @@ root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/contain
+We have prepared a sample CoCo operator custom resource that is based on the standard `ccruntime.yamlccruntime.yaml`, but in addition has the the decryption keys and configuration required to decrypt this sample container image. To test out creating a workload from the sample encrypted container image, we can take the following steps:
 
+### Swap out the standard custom resource for our sample
+
```
It might be worth adding a line as to why the custom resource needs to be removed. Something like
"Support for multiple custom resources in not available in the current release. Consequently, if a custom resource already exists, then you'll need to remove it first before deploying a new one. "

stevenhorsman replied:
Thanks for the suggestion. I've added it in.

wainersm commented on quickstart.md:
```diff
@@ -174,13 +174,85 @@ root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/contain
 
-* *TBD: based on https://github.com/confidential-containers/operator/issues/77* *
+A demo container image is provided at [docker.io/katadocker/ccv0-ssh](https://hub.docker.com/r/katadocker/ccv0-ssh).
+It is encrypted with [Attestation Agent](https://github.com/confidential-containers/attestation-agent)'s [offline file system key broker](https://github.com/confidential-containers/attestation-agent/tree/64c12fbecfe90ba974d5fe4896bf997308df298d/src/kbc_modules/offline_fs_kbc) and [`aa-offline_fs_kbc-keys.json`](./aa-offline_fs_kbc-keys.json) as its key file.
```
I think it should like to https://github.com/confidential-containers/documentation/blob/main/demos/ssh-demo/aa-offline_fs_kbc-keys.json rather than `./aa-offline_fs_kbc-keys.json`.

Other than that, this PR is ready to go. Thanks again @stevenhorsman !

bpradipt reviewed (approved):
/lgtm
Thanks @stevenhorsman

fitzthum reviewed (approved):
See comment for one clarification we might add. Otherwise looks great.

stevenhorsman commented:
> @stevenhorsman please squash the commits into a single one..

Yep, now I've got and addressed all the comments, I've `fixup`'d the commits, so I think this is ready to merge now.
